app/controllers/concerns/aal3_concern.rb

```diff
@@ -6,4 +6,12 @@ def confirm_user_has_aal3_mfa_if_requested
     return if mfa_policy.aal3_mfa_enabled?
     redirect_to two_factor_options_url
   end
+
+  def aal3_redirect_url(user)
+    if TwoFactorAuthentication::PivCacPolicy.new(user).enabled? && !mobile?
+      login_two_factor_piv_cac_url
+    elsif TwoFactorAuthentication::WebauthnPolicy.new(user).enabled?
+      login_two_factor_webauthn_url
+    end
```

Comment:
This needs to default to the `two_factor_options_url` if both of those checks are false.